hack/quickstart-setup.sh

```diff
@@ -241,31 +77,13 @@ if [[ -z "${MGC_WORKLOAD_CLUSTERS_COUNT}" ]]; then
   MGC_WORKLOAD_CLUSTERS_COUNT=1
 fi
 
-cleanup
+cleanupKind
 
-port80=9090
-port443=8445
-proxyPort=9200
-metalLBSubnetStart=200
-
-# Create network for the clusters
-docker network create -d bridge --subnet 172.31.0.0/16 mgc --gateway 172.31.0.1 \
-  -o "com.docker.network.bridge.enable_ip_masquerade"="true" \
-  -o "com.docker.network.driver.mtu"="1500"
-
-# Create Kind control plane cluster
-kindCreateCluster ${KIND_CLUSTER_CONTROL_PLANE} ${port80} ${port443}
-
-# Create Kind workload cluster(s)
-if [[ -n "${MGC_WORKLOAD_CLUSTERS_COUNT}" ]]; then
-  for ((i = 1; i <= ${MGC_WORKLOAD_CLUSTERS_COUNT}; i++)); do
-    kindCreateCluster ${KIND_CLUSTER_WORKLOAD}-${i} $((${port80} + ${i})) $((${port443} + ${i})) $((${i} + 1))
-  done
-fi
+kindSetupMGCClusters ${KIND_CLUSTER_CONTROL_PLANE} ${KIND_CLUSTER_WORKLOAD} ${port80} ${port443} ${MGC_WORKLOAD_CLUSTERS_COUNT}
 
 # Apply Cluster Configurations to Control cluster
 # Deploy OCM hub
-deployOCMHub ${KIND_CLUSTER_CONTROL_PLANE}
+deployOCMHub ${KIND_CLUSTER_CONTROL_PLANE} "minimal"
```

Comment:
What is the purpose of the "minimal" option? What are redis and istio being installed for in the hub currently that they aren't needed here?

Reply:
created #413 to revisit this part